+sig/+node/Signal.m

```diff
@@ -286,53 +286,65 @@
     end
     
     function fs = flattenStruct(this)
-      % Use a struct with signal fields as a blueprint to wire up signals
-      % as inputs to this target so that their values will set the field
-      % values directly in the target's struct value all done in mexnet
-      % according to the transfer opcode
+      % Returns a signal `fs` whose value is a struct containing 
+      % "flattened" fields for the signal fields in the struct that 
```

Comment:
again with the quotes 🐙

Reply:
yes, I will use "" to evoke physical metaphor, as is commonly used in English : )